examples/shaders/shared/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "shared"
```

Comment:
This will probably need to be renamed if you want to publish on crates.io in the future.

Reply:
This is in `examples`, which will not be published.